tasks/dotnettracer.py

```diff
@@ -0,0 +1,146 @@
+"""
+customaction namespaced tasks
+"""
+from __future__ import print_function
+
+import glob
+import os
+import shutil
+import sys
+
+from invoke import task
+from invoke.exceptions import Exit, ParseError, Failure, UnexpectedExit
+from .ssm import get_signing_cert, get_pfx_pass
+
+projects_to_restore = [
+    "Datadog.Trace", "Datadog.Trace.AspNet", "Datadog.Trace.ClrProfiler.Managed", "Datadog.Trace.ClrProfiler.Managed.Core", "Datadog.Trace.ClrProfiler.Managed.Loader"
+]
+@task
+def build(ctx, vstudio_root=None, arch="x64", major_version='7', debug=False):
```

Comment:
If we use `arch` below, we should change `x64` to `All` to build the msi for all supported architectures. 
```suggestion
def build(ctx, vstudio_root=None, arch="All", major_version='7', debug=False):
```